readme.md

```diff
@@ -36,7 +36,7 @@ $ npm install execa
 const execa = require('execa');
 
 (async () => {
-	const {stdout} = await execa('echo', ['unicorns']);
+	const {stdout} = await execa('echo unicorns');
```

Comment:
This is unrelated to the actual change. And even though we support this syntax, I still prefer to show the array version first.

Reply:
Fixed!